### PR TITLE
Update emit.js

### PR DESCRIPTION
**Pull request by chaim-aharonson-applitools**
following CR, since there currently no consumers for `config.visualgrid` its the best time to fix it  (GCTs which support/consume the parameter in PR)

### CODE REVIEW

File: packages/sdk-coverage-tests/src/generate/emit.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ function emitTest(test, {makeSpecEmitter, makeFile}) {
     throw new Error(`Missing implementation for test ${test.name}`)
   }
   test.config.baselineName = test.config.baselineName || test.key
-  test.config.visualgrid = test.vg || false
+  test.vg && (test.config.vg = true)
```
Comment (kyryloonufriiev) [inline packages/sdk-coverage-tests/src/generate/emit.js:29]: On a second thought about that, I think this is not right to add custom stuff in the config object.

**Review by kyryloonufriiev — CHANGES_REQUESTED** [review]
```
test.test.call(utils, {...sdk, config: test.config, env: test.env, meta: test.meta})
```
I think you just need to pass everything from the test object to the test function.

File: packages/sdk-coverage-tests/src/generate/emit.js [hunk 2]
```diff
@@ -45,7 +44,7 @@ function emitTest(test, {makeSpecEmitter, makeFile}) {
   const sdk = makeSpecEmitter(emitter, test)
   test.output = output
   if (test.page) sdk.driver.visit(test.page)
-  test.test.call(utils, {...sdk, config: test.config, env: test.env, meta: test.meta})
+  test.test.call(utils, {...test, ...sdk})
```
Comment (kyryloonufriiev) [inline packages/sdk-coverage-tests/src/generate/emit.js:47]: @chaim-aharonson-applitools What do you think? this is what I propose.

Comment (chaim-aharonson-applitools) [reply packages/sdk-coverage-tests/src/generate/emit.js:47]: great, it works perfectly

**Pull request merged**